javascript-forms/form-values.js

```diff
@@ -0,0 +1,16 @@
+const $contactForm = document.querySelector('#contact-form');
+
+$contactForm.addEventListener('submit', submitFunction);
+
+function submitFunction(event) {
+  event.preventDefault();
+
+  const $formData = {
+    name: $contactForm.elements.name.value,
+    email: $contactForm.elements.email.value,
+    message: $contactForm.elements.message.value,
+  };
+
+  console.log($formData);
+  document.getElementById('contact-form').reset();
```

Comment:
You've already queried for this element on line 1, so you should just utilize that variable rather than querying again.